india_forecast_app/models/pvnet/model.py

```diff
@@ -132,6 +132,9 @@ def _prepare_data_sources(self):
         # Process/cache remote zarr locally
         process_and_cache_nwp(nwp_source_file_path, nwp_path)
 
+        # set NWP_ZARR_PATH as this new path
+        os.environ["NWP_ZARR_PATH"] = nwp_path
```

Comment:
OCF-datapipes should be reading the zarr from the config on L162, rather than from the env var

Reply:
you got a link for that?

Reply:
you are right, let me remove that